Makefile

```diff
@@ -89,36 +89,48 @@ ifeq ($(TARGET_ARCH),x64)
 	TARGET_ARCH_DEBIAN = amd64
 endif
 
-TEMPORARY_DIRECTORY = release/.tmp
 APPLICATION_NAME_LOWERCASE = $(shell echo $(APPLICATION_NAME) | tr A-Z a-z)
 APPLICATION_VERSION_DEBIAN = $(shell echo $(APPLICATION_VERSION) | tr "-" "~")
 
 # ---------------------------------------------------------------------
 # Rules
 # ---------------------------------------------------------------------
 
-release/electron-$(TARGET_PLATFORM)-$(TARGET_ARCH)-dependencies/node_modules: package.json npm-shrinkwrap.json
+release:
+	mkdir -p $@
+
+release/.tmp: | release
+	mkdir -p $@
+
+release/electron-$(TARGET_PLATFORM)-$(TARGET_ARCH)-dependencies: | release
+	mkdir -p $@
```

Comment:
No need for the `-p` flags, because the Makefile dependency rules mean the directories will always get created in the right order :-)